azurefunctions-extensions-bindings-blob/azurefunctions/extensions/bindings/blob/utils.py

```diff
@@ -0,0 +1,35 @@
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License.
+import os
+from pydantic import BaseModel, Field, field_validator
+
+
+class ConnectionConfig(BaseModel):
+    connection_string: str = Field(
+        ..., description="Storage account connection string name"
+    )
+
+    @field_validator("connection_string")
```

Comment:
This decorator is only used for throwing a validation error? What's the difference if we directly throw a validation error instead of a Value error in this  function?